### DISCUSSION TITLE
2006/01/10/chocolate-benchmarks

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#4] [Announcements]
# 2006/01/10/chocolate-benchmarks

http://localhost:4000/2006/01/10/chocolate-benchmarks

**Comment by jengalas** [comment]
Posted by Dick Merrill
October 27, 2013 at 8:58 am

_[Imported from WordPress]_

I have been visiting some airway beacon sites in the Eastern US and have documented them in a short report. I would like to share them with you, if you will tell me how.

Dick Merrill
Chuckey, TN

**Reply by jengalas** [reply]
I have sent you an e-mail reply. I’m looking forward to seeing your report!
